### DISCUSSION TITLE
ERR_SSL_KEY_USAGE_INCOMPATIBLE

### DISCUSSION THREAD

**Discussion by MMStandley** [Security-Onion-Solutions/securityonion#12330] [2.4] [answered]
### Version

2.4.50

### Installation Method

Network installation on Red Hat derivative like Oracle, Rocky, Alma, etc.

### Description

other (please provide detail below)

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

8

### RAM

128gb

### Storage for /

1tb

### Storage for /nsm

1tb

### Network Traffic Collection

span port

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

When trying to access the web interface for SO, I get an ERR_SSL_KEY_USAGE_INCOMPATIBLE error message. 
Neither Edge or Chrome works. So I followed the instructions from a previous discussion: https://github.com/Security-Onion-Solutions/securityonion/discussions/11803
But this did not fix my issue. I did discover that using Firefox works.

Does anybody have any advice on this? Nothing had changed as far as I could tell, so I don't even have anything to point fingers at right now.

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by robbiemarshall** [comment]
What version of Security Onion are you running?  Your post says 2.4.50, but that version is not out yet.

**Reply by MMStandley** [reply]
Sorry, I don't know where the extra 0 came from, it's 2.4.5

**Reply by robbiemarshall** [reply, marked as answer]
You need to run Soup and update.  The cert error is a known bug that was corrected in later versions of Security Onion.

**Reply by samuel809** [reply]
what browser are you using?
test firefox...

**Reply by samuel809** [reply]
its a certificate issue...
